Mathlib/Probability/Kernel/Composition/IntegralCompProd.lean

```diff
@@ -261,4 +278,185 @@ theorem setIntegral_compProd_univ_left (f : β × γ → E) {t : Set γ} (ht : M
     ∫ z in univ ×ˢ t, f z ∂(κ ⊗ₖ η) a = ∫ x, ∫ y in t, f (x, y) ∂η (a, x) ∂κ a := by
   simp_rw [setIntegral_compProd MeasurableSet.univ ht hf, Measure.restrict_univ]
 
+end compProd
+
+section comp
+
+variable {κ : Kernel α β} {η : Kernel β γ}
+
+theorem _root_.MeasureTheory.StronglyMeasurable.integral_kernel [NormedSpace ℝ E] ⦃f : β → E⦄
```

Comment:
Can you move this lemma to the file Mathlib.Probability.Kernel.MeasurableIntegral, where `StronglyMeasurable.integral_kernel_prod_right` and friends are defined?